### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justinegeffen**
First pass review; formatting and punctuation.

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,89 +23,85 @@ This repository is licensed under the [Mattermost Source Available License](LICE
 
 ## Overview
 
-This plugin supports a two-way integration between Mattermost and Microsoft
-Outlook Calendar. For a stable production release, please download the latest
-version [in the Releases
-tab](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and
-follow [these instructions](#configuration) for install and configuration.
+This plugin supports a two-way integration between Mattermost and Microsoft Outlook Calendar. For a stable production release, please download the latest version [in the Releases tab](https://github.com/mattermost/mattermost-plugin-mscalendar/releases) and follow [these instructions](#configuration) for install and configuration.
```
Comment (aaronrothschild) [inline README.md:26]: Shouldn't we be directing people to the PLugin Marketplace to download this?

**Review by aaronrothschild — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -23,89 +23,85 @@ This repository is licensed under the [Mattermost Source Available License](LICE
...
 
 ### Step 2: Configure Plugin Settings
 
-Copy the `Client ID` and `Tenant ID` from the Azure portal
+1. Copy the `Client ID` and `Tenant ID` from the Azure portal.
 
 <img width="500" src="https://user-images.githubusercontent.com/6913320/76779336-9109c480-6781-11ea-8cde-4b79e5b2f3cd.png"/>
 
-**`System Console` > `PLUGINS` > `Microsoft Calendar`**
+2. Navigate to **System Console > PLUGINS (Beta > Microsoft Calendar**. Fill in the following fields:
```
Comment (mickmister) [inline README.md:102]:
```suggestion
2. Navigate to **System Console > PLUGINS (BETA) > Microsoft Calendar**. Fill in the following fields:
```

**Review by mickmister — APPROVED** [review]
Thanks for addressing the feedback @justinegeffen! I just have one more request. Approving preemptively

**Comment by justinegeffen** [comment]
@aaronrothschild, following up on this for your approval.

**Review by aaronrothschild — APPROVED** [review]

**Comment by hanzei** [comment]
@justinegeffen Is this ready to be merged?

**Comment by justinegeffen** [comment]
Yep, this PR is good to go I reckon. I'll move it over to Gitbook next.

**Pull request merged**